### DISCUSSION TITLE
Traxxas ESC giving a trim centering error

### DISCUSSION THREAD

**Discussion by legop3** [iNavFlight/inav#10044] [Q&A]
Hi, I am building an inav rover based on an existing RC car that I clobbered together. This car has a traxxas vxl-3s ESC, and no matter what I do I cannot get it to calibrate to my flight controller. I have tried different ranges and frequencies, reversible motor mode, not reversible motor mode, etc. I also can't seem to be able to enter motor test mode in configurator when I have reversible motors mode turned on. Thanks for any help.

**Comment by cyrilius42** [comment]
not sure if youve figured this out, but im having an identical issue. Found a few threads on the FPV rc group over on facebook which guides you to a github post with a decent workaround and some other fiddly stuff. Ive yet to try it myself but will attempt and report back

https://github.com/iNavFlight/inav/issues/9175?fbclid=IwY2xjawKd5IBleHRuA2FlbQIxMABicmlkETFwVGM2SmE1SEtNTTcySDkzAR7YPAWlxG-00W8-N1-N89A8JBQ1rzu3JCHC0F1DonlfiW_ScIFonr9f7Vxqlw_aem_ZHoQrH--pccs-sHdeLBl1A

**Reply by cyrilius42** [reply]
https://www.facebook.com/share/p/1BX59u2DNt/ this is the original post i found the link from. Ive even called traxxas support asking about what ppm signals the esc is expecting etc etc..... didnt help much lol.
